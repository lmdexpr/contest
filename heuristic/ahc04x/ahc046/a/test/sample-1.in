20 40
3 7
14 16
19 19
0 8
0 10
17 11
11 7
19 12
10 12
5 4
13 5
19 10
19 14
1 19
8 5
6 9
9 3
10 0
8 7
17 17
4 2
1 13
9 17
13 4
9 14
19 13
16 9
8 17
10 18
14 3
14 13
18 19
3 14
8 13
7 9
12 18
8 6
10 7
12 1
5 17
