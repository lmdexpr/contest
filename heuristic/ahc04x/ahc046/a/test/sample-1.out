S D
M U
A D
M U
M U
M U
M U
S R
M L
M L
M L
S D
S R
S L
S U
M R
M R
